packages/graphback-datasync/src/providers/DatasyncMongoDBDataProvider.ts

```diff
@@ -134,6 +134,8 @@ export class DataSyncMongoDBDataProvider<Type = any> extends MongoDBDataProvider
         queryResult[fieldNames.updatedAt] !== undefined &&
         clientData[fieldNames.updatedAt].toString() !== queryResult[fieldNames.updatedAt].toString()
```

Comment:
Client side will always will be string as it comes thru scalar.

Reply:
Should the fix be done here https://github.com/aerogear/graphback/blob/master/packages/graphback-core/src/utils/fieldTransformHelpers.ts?